Crash when navigating the homescreen

### Describe the bug

1. Update app
2. Log in to existing user
3. Scroll around
4. ???
5. Crash

.. not super helpful but it the app + shield completely froze and upon unfreezing the Jellyfin app crashed

### Logs

---
client: Jellyfin for Android TV
client_version: 0.14.0-beta.1
client_repository: https://github.com/jellyfin/jellyfin-androidtv
type: crash_report
format: markdown
---
### Logs

***Stack Trace***: 
```log
java.lang.RuntimeException: Unable to resume activity {org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3835)
	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3867)
	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loop(Looper.java:193)
	at android.app.ActivityThread.main(ActivityThread.java:6721)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
	at org.jellyfin.androidtv.ui.itemhandling.BaseRowItem.refresh(BaseRowItem.java:496)
	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.refreshCurrentItem(StdRowsFragment.java:216)
	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.onResume(StdRowsFragment.java:89)
	at org.jellyfin.androidtv.ui.home.HomeFragment.onResume(HomeFragment.kt:69)
	at androidx.fragment.app.Fragment.performResume(Fragment.java:3180)
	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:606)
	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1424)
	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2968)
	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:2900)
	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:334)
	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:323)
	at android.app.Activity.performResume(Activity.java:7360)
	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3827)
	... 11 more

```  
***Logcat***: 
```log
--------- beginning of main
07-18 06:52:53.827 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"ForceKeepAlive","MessageId":"00000000000000000000000000000000","Data":60}
07-18 06:52:53.831 I/KeepAliveHelper(24578): Using a KeepAlive message delay of 30 seconds
07-18 06:52:53.834 I/OkHttpWebsocketSession(24578): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
07-18 06:52:53.856 D/SocketInstance(24578): Credential update requested
07-18 06:52:53.857 D/SocketInstance(24578): Unable to update credentials: credentials did not change
07-18 06:52:53.862 D/StartupActivity(24578): Opening next activity Intent { flg=0x10004000 cmp=org.jellyfin.androidtv/.ui.browsing.MainActivity }
07-18 06:52:53.872 D/ACRA    (24578): onActivityPaused class org.jellyfin.androidtv.ui.startup.StartupActivity
07-18 06:52:53.882 W/ActivityThread(24578): handleWindowVisibility: no activity for token android.os.BinderProxy@3b4acff
07-18 06:52:53.886 D/ACRA    (24578): onActivityCreated class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:52:53.899 I/BackgroundService(24578): Adding BackgroundServiceFragment to activity
07-18 06:52:53.901 D/ACRA    (24578): onActivityStarted class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:52:53.916 I/BackgroundService(24578): Adding BackgroundServiceFragment to activity
07-18 06:52:53.917 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"dfca1e3ae6354a83af53f2d158f82aec"}
07-18 06:52:53.920 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Views?format=json
07-18 06:52:53.923 D/MediaManager(24578): Added event listener.  Total listeners: 1
07-18 06:52:53.941 D/ACRA    (24578): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:52:53.942 I/HomeFragment(24578): Updating audio queue in HomeFragment (onResume)
07-18 06:52:53.942 D/BackgroundServiceFragment(24578): Restoring background drawable
07-18 06:52:53.943 D/BackgroundServiceFragment(24578): Restoring background drawable
07-18 06:52:53.968 E/libnvcpl_vendor(24578): NvCplGetAppProfileSettingInt failed to get INvCplHalService
07-18 06:52:53.968 E/libnvcpl_vendor(24578): NvCplGetAppProfileSettingInt failed to get INvCplHalService
07-18 06:52:54.264 I/lyfin.androidt(24578): Background concurrent copying GC freed 196454(13MB) AllocSpace objects, 7(260KB) LOS objects, 49% free, 13MB/27MB, paused 2.846ms total 137.214ms
07-18 06:52:54.377 I/KtorClient(24578): GET https://jellyfin.codeh.tech/Playback/BitrateTest?size=6000000
07-18 06:52:54.410 D/ACRA    (24578): onActivityStopped class org.jellyfin.androidtv.ui.startup.StartupActivity
07-18 06:52:54.412 D/ACRA    (24578): onActivityDestroyed class org.jellyfin.androidtv.ui.startup.StartupActivity
07-18 06:52:54.662 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Views?format=json
07-18 06:52:54.723 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Views?format=json
07-18 06:52:54.726 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items?SortBy=DatePlayed&EnableTotalRecordCount=false&ExcludeLocationTypes=Virtual&MediaTypes=Video&ImageTypeLimit=1&recursive=true&Filters=IsResumable&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&sortOrder=Descending&Limit=50&format=json
07-18 06:52:54.727 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2130c095-606e-4961-9adc-bacfbdabe373&Limit=50&ImageTypeLimit=1&format=json
07-18 06:52:54.729 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=767bffe4f11c93ef34b805451a696a4e&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:54.731 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=f137a2dd21bbc1b99aa5c0f6bf02a805&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:54.732 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=fe68e648dcdb7f21053e08c8a036517c&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:54.733 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=ec27261a0c809aa017c1c9e0d02c31ac&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:54.734 D/jellyfin-apiclient(24578): Adding request to queue: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=772db055fe30a332896493a09158bb2c&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:54.800 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Views?format=json
07-18 06:52:54.835 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:54.900 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:55.091 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=f137a2dd21bbc1b99aa5c0f6bf02a805&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:55.194 I/KtorClient(24578): GET https://jellyfin.codeh.tech/Playback/BitrateTest?size=8000000
07-18 06:52:55.202 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items?SortBy=DatePlayed&EnableTotalRecordCount=false&ExcludeLocationTypes=Virtual&MediaTypes=Video&ImageTypeLimit=1&recursive=true&Filters=IsResumable&Fields=PrimaryImageAspectRatio%2COverview%2CItemCounts%2CDisplayPreferencesId%2CChildCount&sortOrder=Descending&Limit=50&format=json
07-18 06:52:55.222 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:55.283 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:55.693 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=fe68e648dcdb7f21053e08c8a036517c&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:55.749 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=ec27261a0c809aa017c1c9e0d02c31ac&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:55.816 I/KtorClient(24578): GET https://jellyfin.codeh.tech/Playback/BitrateTest?size=10000000
07-18 06:52:56.040 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=767bffe4f11c93ef34b805451a696a4e&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:52:56.057 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:56.081 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:57.043 I/AutoBitrate(24578): Auto bitrate set to: 65146579
07-18 06:52:57.519 I/KtorClient(24578): GET https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Views?includeHidden=false
07-18 06:52:57.643 I/lyfin.androidt(24578): Method exceeds compiler instruction limit: 21432 in org.jellyfin.sdk.model.api.BaseItemDto org.jellyfin.sdk.model.api.BaseItemDto$$serializer.deserialize(kotlinx.serialization.encoding.Decoder)
07-18 06:52:57.730 I/WM-WorkerWrapper(24578): Worker result SUCCESS for Work [ id=e7953f1b-d98e-4a60-bd0b-168f8288bdd3, tags={ org.jellyfin.androidtv.integration.LeanbackChannelWorker } ]
07-18 06:52:57.763 D/Volley  (24578): [20326] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] https://jellyfin.codeh.tech/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2130c095-606e-4961-9adc-bacfbdabe373&Limit=50&ImageTypeLimit=1&format=json 0x2aa09daa NORMAL 4> [lifetime=3031], [size=84477], [rc=200], [retryCount=0]
07-18 06:52:57.765 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Shows/NextUp?Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&UserId=2130c095-606e-4961-9adc-bacfbdabe373&Limit=50&ImageTypeLimit=1&format=json
07-18 06:52:57.788 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:52:57.811 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:01.931 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:02.796 D/Volley  (24578): [20325] NetworkUtility.logSlowRequests: HTTP response for request=<[ ] https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=772db055fe30a332896493a09158bb2c&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json 0x2aa09daa NORMAL 9> [lifetime=7102], [size=25605], [rc=200], [retryCount=0]
07-18 06:53:02.798 I/jellyfin-apiclient(24578): Response received from: https://jellyfin.codeh.tech/Users/2130c095-606e-4961-9adc-bacfbdabe373/Items/Latest?ParentId=772db055fe30a332896493a09158bb2c&Fields=PrimaryImageAspectRatio%2COverview%2CChildCount&Limit=50&GroupItems=true&format=json
07-18 06:53:03.112 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:03.119 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:07.693 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:07.725 I/chatty  (24578): uid=10079(org.jellyfin.androidtv) identical 3 lines
07-18 06:53:07.733 E/ThemeUtils(24578): View class org.jellyfin.androidtv.ui.AsyncImageView is an AppCompat widget that can only be used with a Theme.AppCompat theme (or descendant).
07-18 06:53:08.765 I/lyfin.androidt(24578): NativeAlloc concurrent copying GC freed 873(171KB) AllocSpace objects, 0(0B) LOS objects, 48% free, 25MB/49MB, paused 191us total 508.818ms
07-18 06:53:08.959 I/lyfin.androidt(24578): NativeAlloc concurrent copying GC freed 1043(186KB) AllocSpace objects, 0(0B) LOS objects, 48% free, 25MB/49MB, paused 167us total 191.310ms
07-18 06:53:09.159 W/InputEventSender(24578): Failed to send key event on channel 'ClientState{8e5d715 uid 10079 pid 24578} (server)'.  status=-32
07-18 06:53:09.159 W/InputMethodManager(24578): Unable to send input event to IME: com.google.android.leanback.ime/com.google.leanback.ime.LeanbackImeService dropping: KeyEvent { action=ACTION_DOWN, keyCode=KEYCODE_DPAD_DOWN, scanCode=108, metaState=0, flags=0x8, repeatCount=0, eventTime=1189453240, downTime=1189453240, deviceId=7, source=0x301 }
07-18 06:53:09.609 I/lyfin.androidt(24578): NativeAlloc concurrent copying GC freed 3134(324KB) AllocSpace objects, 0(0B) LOS objects, 48% free, 25MB/49MB, paused 920us total 365.081ms
07-18 06:53:23.841 I/OkHttpWebsocketSession(24578): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
07-18 06:53:23.921 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"ef7d0252cf894bc8808c5edbdc83d091"}
07-18 06:53:26.696 I/AssistStructure(24578): Flattened final assist data: 11236 bytes, containing 1 windows, 203 views
07-18 06:53:27.988 I/AssistStructure(24578): Flattened final assist data: 11236 bytes, containing 1 windows, 203 views
07-18 06:53:53.842 I/OkHttpWebsocketSession(24578): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
07-18 06:53:53.903 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"a23bf19ae50e4eba97bdd6fd75cabc9d"}
07-18 06:54:07.773 I/Choreographer(24578): Skipped 34 frames!  The application may be doing too much work on its main thread.
07-18 06:54:23.844 I/OkHttpWebsocketSession(24578): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
07-18 06:54:23.902 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"481db3db03e74aec9816c60611ff3af5"}
07-18 06:54:53.788 D/ACRA    (24578): onActivityPaused class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:54:53.845 I/OkHttpWebsocketSession(24578): Sending (raw) message {"MessageType":"KeepAlive","Data":{}}
07-18 06:54:53.846 D/ACRA    (24578): onActivityStopped class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:54:53.848 D/ACRA    (24578): onActivitySaveInstanceState class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:54:53.905 I/OkHttpWebsocketSession(24578): Receiving (raw) message {"MessageType":"KeepAlive","MessageId":"cfd1576434574f76ab45e3fba57453e9"}
07-18 06:54:57.528 D/ACRA    (24578): onActivityStarted class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:54:57.538 I/JellyfinApplication$onCreate(24578): Process lifecycle started
07-18 06:54:57.548 D/SessionRepositoryImpl(24578): Restoring session
07-18 06:54:57.559 D/SessionRepositoryImpl(24578): Destroying current session
07-18 06:54:57.560 D/ACRA    (24578): onActivityResumed class org.jellyfin.androidtv.ui.browsing.MainActivity
07-18 06:54:57.569 D/StdRowsFragment(24578): Refresh item "Superwho? - Superwho? / Super-héros malgré lui (2022) - Trailer (English subs)"
07-18 06:54:57.582 D/AndroidRuntime(24578): Shutting down VM
--------- beginning of crash
07-18 06:54:57.586 E/AndroidRuntime(24578): FATAL EXCEPTION: main
07-18 06:54:57.586 E/AndroidRuntime(24578): Process: org.jellyfin.androidtv, PID: 24578
07-18 06:54:57.586 E/AndroidRuntime(24578): java.lang.RuntimeException: Unable to resume activity {org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3835)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3867)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.os.Handler.dispatchMessage(Handler.java:106)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.os.Looper.loop(Looper.java:193)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.ActivityThread.main(ActivityThread.java:6721)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at java.lang.reflect.Method.invoke(Native Method)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
07-18 06:54:57.586 E/AndroidRuntime(24578): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at org.jellyfin.androidtv.ui.itemhandling.BaseRowItem.refresh(BaseRowItem.java:496)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.refreshCurrentItem(StdRowsFragment.java:216)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.onResume(StdRowsFragment.java:89)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at org.jellyfin.androidtv.ui.home.HomeFragment.onResume(HomeFragment.kt:69)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.Fragment.performResume(Fragment.java:3180)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:606)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1424)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2968)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:2900)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:334)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:323)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.Activity.performResume(Activity.java:7360)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3827)
07-18 06:54:57.586 E/AndroidRuntime(24578): 	... 11 more
07-18 06:54:57.586 E/ACRA    (24578): ACRA caught a RuntimeException for org.jellyfin.androidtv
07-18 06:54:57.586 E/ACRA    (24578): java.lang.RuntimeException: Unable to resume activity {org.jellyfin.androidtv/org.jellyfin.androidtv.ui.browsing.MainActivity}: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3835)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.ActivityThread.handleResumeActivity(ActivityThread.java:3867)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.servertransaction.ResumeActivityItem.execute(ResumeActivityItem.java:51)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.servertransaction.TransactionExecutor.executeLifecycleState(TransactionExecutor.java:145)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:70)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1809)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.os.Handler.dispatchMessage(Handler.java:106)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.os.Looper.loop(Looper.java:193)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.ActivityThread.main(ActivityThread.java:6721)
07-18 06:54:57.586 E/ACRA    (24578): 	at java.lang.reflect.Method.invoke(Native Method)
07-18 06:54:57.586 E/ACRA    (24578): 	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:493)
07-18 06:54:57.586 E/ACRA    (24578): 	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
07-18 06:54:57.586 E/ACRA    (24578): Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'java.util.UUID org.jellyfin.sdk.model.api.UserDto.getId()' on a null object reference
07-18 06:54:57.586 E/ACRA    (24578): 	at org.jellyfin.androidtv.ui.itemhandling.BaseRowItem.refresh(BaseRowItem.java:496)
07-18 06:54:57.586 E/ACRA    (24578): 	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.refreshCurrentItem(StdRowsFragment.java:216)
07-18 06:54:57.586 E/ACRA    (24578): 	at org.jellyfin.androidtv.ui.browsing.StdRowsFragment.onResume(StdRowsFragment.java:89)
07-18 06:54:57.586 E/ACRA    (24578): 	at org.jellyfin.androidtv.ui.home.HomeFragment.onResume(HomeFragment.kt:69)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.Fragment.performResume(Fragment.java:3180)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentStateManager.resume(FragmentStateManager.java:606)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentStateManager.moveToExpectedState(FragmentStateManager.java:285)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentStore.moveToExpectedState(FragmentStore.java:113)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentManager.moveToState(FragmentManager.java:1424)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentManager.dispatchStateChange(FragmentManager.java:2968)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentManager.dispatchResume(FragmentManager.java:2900)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentController.dispatchResume(FragmentController.java:285)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentActivity.onResumeFragments(FragmentActivity.java:334)
07-18 06:54:57.586 E/ACRA    (24578): 	at androidx.fragment.app.FragmentActivity.onPostResume(FragmentActivity.java:323)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.Activity.performResume(Activity.java:7360)
07-18 06:54:57.586 E/ACRA    (24578): 	at android.app.ActivityThread.performResumeActivity(ActivityThread.java:3827)
07-18 06:54:57.586 E/ACRA    (24578): 	... 11 more
07-18 06:54:57.586 D/ACRA    (24578): Building report
07-18 06:54:57.591 D/ACRA    (24578): Starting collectors with priority FIRST
07-18 06:54:57.592 D/ACRA    (24578): Calling collector org.acra.collector.DropBoxCollector
07-18 06:54:57.592 D/ACRA    (24578): Collector org.acra.collector.DropBoxCollector completed
07-18 06:54:57.593 D/ACRA    (24578): Calling collector org.acra.collector.StacktraceCollector
07-18 06:54:57.595 D/ACRA    (24578): Collector org.acra.collector.StacktraceCollector completed
07-18 06:54:57.595 D/ACRA    (24578): Calling collector org.acra.collector.LogCatCollector
07-18 06:54:57.639 D/ACRA    (24578): Retrieving logcat output (buffer:default)...

```  

### App information

***App version***: `0.14.0-beta.1` (`140001`)  
***Package name***: `org.jellyfin.androidtv`  
***Build***: 
```json
{"BOARD":"sif","BOOTLOADER":"32.00.2019.50-t210-79558a05","BRAND":"NVIDIA","CPU_ABI":"armeabi-v7a","CPU_ABI2":"armeabi","DEVICE":"sif","DISPLAY":"PPR1.180610.011.4079180_2740.7538","FINGERPRINT":"NVIDIA\/sif\/sif:9\/PPR1.180610.011\/4079180_2740.7538:user\/release-keys","HARDWARE":"sif","HOST":"mobile-u64-5495-d6000","ID":"PPR1.180610.011","IS_DEBUGGABLE":false,"IS_EMULATOR":false,"MANUFACTURER":"NVIDIA","MODEL":"SHIELD Android TV","PERMISSIONS_REVIEW_REQUIRED":false,"PRODUCT":"sif","RADIO":"unknown","SUPPORTED_32_BIT_ABIS":["armeabi-v7a","armeabi"],"SUPPORTED_64_BIT_ABIS":[],"SUPPORTED_ABIS":["armeabi-v7a","armeabi"],"TAGS":"release-keys","TIME":1621883973000,"TYPE":"user","UNKNOWN":"unknown","USER":"NVIDIA","VERSION":{"ACTIVE_CODENAMES":[],"BASE_OS":"","CODENAME":"REL","FIRST_SDK_INT":28,"INCREMENTAL":"4079180_2740.7538","PREVIEW_SDK_INT":0,"RELEASE":"9","RESOURCES_SDK_INT":28,"SDK":"28","SDK_INT":28,"SECURITY_PATCH":"2021-04-05"}}
```  
***Build config***: 
```json
{"APPLICATION_ID":"org.jellyfin.androidtv","BUILD_TYPE":"release","DEBUG":false,"DEVELOPMENT":false,"VERSION_CODE":140001,"VERSION_NAME":"0.14.0-beta.1"}
```  

### Device information

***Android version***: `9`  
***Device brand***: `NVIDIA`  
***Device product***: `sif`  
***Device model***: `SHIELD Android TV`  

### Crash information

***Start time***: `2022-07-18T06:52:33.388-06:00`  
***Crash time***: `2022-07-18T06:54:57.751-06:00`  



### Application version

0.14.0-beta.1

### Where did you install the app from?

Google Play

### Device information

Nvidia Shield Pro (2019)

### Android version

Android 9

### Jellyfin server version

10.8.1

--- a/app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/BaseRowItem.java
+++ b/app/src/main/java/org/jellyfin/androidtv/ui/itemhandling/BaseRowItem.java
@@ -33,6 +33,7 @@ import java.text.SimpleDateFormat;
 import java.util.Arrays;
 
 import kotlin.Lazy;
+import timber.log.Timber;
 
 public class BaseRowItem {
     private int index;
@@ -493,13 +494,19 @@ public class BaseRowItem {
     public void refresh(final EmptyResponse outerResponse) {
         switch (type) {
             case BaseItem:
-                apiClient.getValue().GetItemAsync(getItemId(), KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue().getId().toString(), new Response<BaseItemDto>() {
-                    @Override
-                    public void onResponse(BaseItemDto response) {
-                        baseItem = response;
-                        outerResponse.onResponse();
-                    }
-                });
+                String id = getItemId();
+                UserDto user = KoinJavaComponent.<UserRepository>get(UserRepository.class).getCurrentUser().getValue();
+                if (Utils.isNonEmpty(id) && user != null) {
+                    apiClient.getValue().GetItemAsync(id, user.getId().toString(), new Response<BaseItemDto>() {
+                        @Override
+                        public void onResponse(BaseItemDto response) {
+                            baseItem = response;
+                            outerResponse.onResponse();
+                        }
+                    });
+                } else {
+                    Timber.w("BaseRowItem.refresh() failed!");
+                }
                 break;
             case Person:
             case Chapter:

